src/transformer.ts

```diff
@@ -560,3 +561,81 @@ const correctHTMLDocHierarchy = (doc: DOM.Document) => {
     }
 };
 
+const XMLNS_URI = 'http://www.w3.org/2000/xmlns/';
+
+declare const document: DOM.Document | undefined;
+
+/**
+ * This addresses a difference in how Firefox treats XML namespace declarations
```

Comment:
Enketo Core relies on namespaces being being copied onto the first child of the main instance.